css/style.css

```diff
@@ -220,34 +220,52 @@ main {
 }
 
 .toggle-switch {
-  display: flex;
+  display: inline-flex;
   align-items: center;
   justify-content: space-around;
   height: 30px;
-  width: 70px;
-  margin-left: 10px;
+  width: 55px;
+  margin-left: 5px;
   background-color: var(--secondary-color);
-  border-radius: 2px;
+  border-radius: 5px;
+  position: relative;
 }
 
-input[type="checkbox"] {
-  -moz-appearance: none;
-  margin: 0;
+.toggle-switch input{
+  display: none;
```

Comment:
If you make the opacity not 0 can you see it? Style it like the switch instead?

Also I think the active is on the wrong element: `.toggle-switch:active` instead should be: `.toggle-switch input:active`

Something like:
```
.toggle-switch input {
  -moz-appearance: none;
}

.toggle-switch input:checked::before {
  ...
}

.toggle-switch input::after {
  ...
}

.toggle-switch input:checked {
  ...
}

.toggle-switch input:checked::after {
  ...
}

.toggle-switch input:active {
  border:1px solid var(--button-active-border-color);
}
```